metron-platform/metron-solr/metron-solr-common/src/test/java/org/apache/metron/solr/integration/components/SolrComponent.java

```diff
@@ -149,19 +149,16 @@ public String getZookeeperUrl() {
   public void addCollection(String name, String configPath)
       throws InterruptedException, IOException, KeeperException, SolrServerException {
     miniSolrCloudCluster.uploadConfigSet(new File(configPath).toPath(), name);
-    CollectionAdminRequest.createCollection(name, 1, 1)
+    CollectionAdminRequest.createCollection(name, name,1, 1)
```

Comment:
Do you know how the second parameter `config` is used by Solr?  What are the ramifications of using just the `name` as the `config` here?

[Javadocs](https://lucene.apache.org/solr/7_3_0/solr-solrj/org/apache/solr/client/solrj/request/CollectionAdminRequest.html#createCollection-java.lang.String-java.lang.String-int-int-) say the following, but I don't understand the repercussions of this.
```
collection - the collection name
config - the collection config
numShards - the number of shards in the collection
numReplicas - the replication factor of the collection (same as numNrtReplicas)
```

Reply:
Maybe I misunderstood but I thought that was the config name.  I will double check.

Reply:
I went into the source.  It's eventually assigned to a `configName` variable.